src/3d/p5.Renderer3D.Immediate.js

```diff
@@ -177,4 +180,50 @@ p5.Renderer3D.prototype._getColorVertexShader = function(){
   return shaderProgram;
 };
 
-module.exports = p5.Renderer3D;
\ No newline at end of file
+//////////////////////////////////////////////
+//Bezier
+//////////////////////////////////////////////
+
+/**
+ * adds a vertex to be drawn in a custom Shape.
+ * @method bezier
+ * @param  {Number} x1 x-coordinate for the first anchor point
+ * @param  {Number} y1 y-coordinate for the first anchor point
+ * @param  {Number} x2 x-coordinate for the first control point
+ * @param  {Number} y2 y-coordinate for the first control point
+ * @param  {Number} x3 x-coordinate for the second control point
+ * @param  {Number} y3 y-coordinate for the second control point
+ * @param  {Number} x4 x-coordinate for the second anchor point
+ * @param  {Number} y4 y-coordinate for the second anchor point
+ * @return {p5.Renderer3D}   [description]
+ * @TODO implement bezier in 3D
+ * @example
+ * <div>
+ * <code>
+ *background(0, 0, 0);
+ *noFill();
+ *stroke(255);
+ *bezier(250,250,0, 100, 100,0,100,0,0, 0,100,0);
+ * </code>
+ * </div>
+ */
+//this implementation of bezier curve is based on Bernstein polynomial
+p5.prototype.bezier = function(x1, y1, z1, x2, y2, z2, x3, y3, z3, x4, y4, z4) {
```

Comment:
instead of creating the bezier function under the global p5 namespace, consider refactoring under p5.Renderer3D.prototype